jsonminify/index.d.ts

```diff
@@ -3,6 +3,6 @@
 // Definitions by: Dan Homola <https://github.com/no23reason>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function minify(json: string): string;
+declare const minify: (json: string) => string;
```

Comment:
It makes sense now! Thank you very much for your time and sorry it took so many commits... It's fixed now.